Source/JavaScriptCore/runtime/JSCJSValue.h

```diff
@@ -482,7 +482,7 @@ class JSValue {
     static_assert(LowestOfHighBits & NumberTag);
     static_assert(!((LowestOfHighBits>>1) & NumberTag));
 
-    // All non-numeric (bool, null, undefined) immediates have bit 2 set.
+    // All non-numeric (bool, null, undefined) immediates have bit 1 set.
```

Comment:
I think this comment is meant to be interpreted as zero indexed. Documenting that seems reasonable though. Maybe "All non-numeric (bool, null, undefined) immediates have bit 1 (zero indexed) set."?

Reply:
Thank you for the review. However, there are sections in other comments as well where the bit ordering is represented as 0-indexed. So how about making a similar documenting to those part? This would help clarify that the bits are 0-indexed in the content of the subsequent comments as well.

Line 454~458:
```
     * These values have the following properties:
     * - Bit 1 (0-indexed) is set (OtherTag) for all four values, allowing real pointers to be
     *   quickly distinguished from all immediate values, including these invalid pointers.
     * - With bit 3 (0-indexed) masked out (UndefinedTag), Undefined and Null share the
     *   same value, allowing null & undefined to be quickly detected.
```

Line 485 as reviewed.
```
// All non-numeric (bool, null, undefined) immediates have bit 1 (0-indexed) set.
```

The expression `0-indexed` are being used in `Source/JavaScriptCore/runtime/JSBigInt.cpp`